### DISCUSSION TITLE
error: Provisioning Profile doesnt include signing certificate

### DISCUSSION THREAD

**Discussion by taylorjosephgriffin** [fastlane/fastlane#21047] [Q&A]
Hi there,

So essentially, my certs were expired so I deleted them from our cloud storage and then ran `match` which generated new certs and a new provisioning profile and now I am unable to submit any builds to testflight because I am getting the below error.

`error: Provisioning profile "match AppStore com.evolus.PROJECT_NAME" doesn't include signing certificate "Apple Distribution: CERTIFICATE". (in target 'PROJECT_NAME' from project 'PROJECT_NAME')`

I am not sure if I did something wrong when trying to regen new certs/profiles. All of the resources seemed to imply that that was all I needed to do.

**Comment by Immmmmmortal1** [comment]
so i
